github-clone.el

```diff
@@ -180,7 +186,7 @@ DIRECTORY.  Then it prompts to fork the repository and add a
 remote named after the github username to the fork."
   (interactive
    (list (read-from-minibuffer "Url or User/Repo: ")
-         (read-directory-name "Directory: " nil default-directory t)))
+         (read-directory-name "Directory: " github-clone-default-directory nil t)))
```

Comment:
I think maybe this should use as github-clone-default-directory, but still default to default-directory, ie

    (read-directory-name "Directory: " github-clone-default-directory default-directory t)

That way, enter still selects default-directory, but the auto-completion target is inside the default directory suggested?

Alternatively perhaps the right way to do this is to make the default value for github-clone-default-directory to be an alias of the value of default-directory, so it only overrides if specified.